iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="backdrop" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" role="none" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
I can't find this comment in context - can anyone point me in the right
direction?

On Thu, Dec 10, 2015 at 10:46 AM, Christopher Joel <notifications@github.com
> wrote:

> In iron-image.html
> <https://github.com/PolymerElements/iron-image/pull/44#discussion_r47266350>
> :
>
> > @@ -105,9 +114,11 @@
> >
> >    <template>
> >
> > -    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
> > -    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
> > -    <content></content>
> > +    <div id="backdrop" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
> > +    <img id="img" role="none" hidden$="[[_computeImgHidden(sizing)]]">
>
> It seems like what you want is for the shadow host (the iron-image) to be
> the canonical source of ARIA for a reader. If that's true, then the img
> is an implementation detail and should be hidden.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PolymerElements/iron-image/pull/44/files#r47266350>.
>